sanic/models/futures.py

```diff
@@ -50,3 +51,9 @@ class FutureStatic(NamedTuple):
     host: Optional[str]
     strict_slashes: Optional[bool]
     content_type: Optional[bool]
+
+
+class FutureSignal(NamedTuple):
+    handler: SignalHandler
+    event: str
```

Comment:
If possible can we change the order for handler and event? The way to register them using a decorator would alway have the event in first place. Would be more intuitive.

Reply:
I think I used this order to mirror Routes. I'll double check.

Reply:
Keeping this:

```python
class FutureRoute(NamedTuple):
    handler: str
    ...
```